news-bundle/contao/classes/News.php

```diff
@@ -26,261 +21,6 @@ class News extends Frontend
 	 */
 	private static $arrUrlCache = array();
 
-	/**
-	 * Page cache array
-	 * @var array
-	 */
-	private static $arrPageCache = array();
-
-	/**
-	 * Update a particular RSS feed
-	 *
-	 * @param integer $intId
-	 */
-	public function generateFeed($intId)
```

Comment:
@contao/developers Do we really want to remove all these methods? None of them are deprecated in Contao 4.13.